tools/phpunit/bootstrap.php

```diff
@@ -102,9 +104,9 @@ static function () use ( $load_plugin, $plugin_name ): void {
 	tests_add_filter(
 		'plugins_loaded',
 		static function () use ( $plugin_name ): void {
-			require_once TESTS_PLUGIN_DIR . '/plugins/' . $plugin_name . '/includes/admin/load.php';
-			require_once TESTS_PLUGIN_DIR . '/plugins/' . $plugin_name . '/includes/admin/server-timing.php';
-			require_once TESTS_PLUGIN_DIR . '/plugins/' . $plugin_name . '/includes/admin/plugins.php';
+			require_once TESTS_REPO_ROOT_DIR . '/plugins/' . $plugin_name . '/includes/admin/load.php';
+			require_once TESTS_REPO_ROOT_DIR . '/plugins/' . $plugin_name . '/includes/admin/server-timing.php';
+			require_once TESTS_REPO_ROOT_DIR . '/plugins/' . $plugin_name . '/includes/admin/plugins.php';
```

Comment:
See above:
```suggestion
			require_once TESTS_PLUGIN_DIR . '/includes/admin/load.php';
			require_once TESTS_PLUGIN_DIR . '/includes/admin/server-timing.php';
			require_once TESTS_PLUGIN_DIR . '/includes/admin/plugins.php';
```

Reply:
Actually we can't do this. See https://github.com/WordPress/performance/pull/1804#discussion_r1932815522

Reply:
What do you mean? These files don't require any dependencies.

Reply:
Plugin dependencies (i.e. Image Prioritizer depending on Optimization Detective)

Reply:
I don't understand. Specific files are being loaded here that only matter for `performance-lab`.

Reply:
Oh right. At least these should be wrapped in a condition so they only are loaded if the `$plugin_name` is `performance-lab`

Reply:
Oh, but that already it the case. This logic is wrapped in:

```php
if ( 'performance-lab' === $plugin_name ) {
```